src/cursor/sys/windows.rs

```diff
@@ -14,7 +14,13 @@ use lazy_static::lazy_static;
 
 use crate::utils::Result;
 
-pub(crate) fn get_cursor_position() -> Result<(u16, u16)> {
+lazy_static! {
+    static ref SAVED_CURSOR_POS: Mutex<Option<(i16, i16)>> = Mutex::new(None);
+}
+
+/// Returns the cursor position (column, row).
```

Comment:
Empty doc comment line between these two lines. Also I'd use the same wording from the previous comment:

Top left cell is represented as `0, 0`.

Short and clear enough.